Source/Engine/Core/Config/LayersAndTagsSettings.cs

```diff
@@ -30,6 +31,29 @@ public static string[] GetCurrentLayers()
             return GetCurrentLayers(out int _);
         }
 
+        /// <summary>
+        /// The layers names.
+        /// </summary>
+        [EditorOrder(10), EditorDisplay("Terrain Layers", EditorDisplayAttribute.InlineStyle), Collection(CanResize = false, Display = CollectionAttribute.DisplayType.Inline)]
+        public string[] TerrainLayers = new string[8];
+
+        /// <summary>
+        /// Gets the current terrain layer names. Returns "Layer" + index for layers without a name.
+        /// </summary>
+        /// <returns>The layer names.</returns>
+        public static string[] GetCurrentTerrainLayers()
+        {
+            string[] layerNames = GameSettings.Load<LayersAndTagsSettings>().TerrainLayers.ToArray();
```

Comment:
I think the ToArray() call here isn't needed as `TerrainLayers` is already a string array

Reply:
Done in https://github.com/FlaxEngine/FlaxEngine/commit/75d4a110de45b5494fd7ee3c1f49597797dd5605